miden-lib/asm/scripts/ASWAP.masm

```diff
@@ -0,0 +1,52 @@
+use.miden::sat::note
+use.miden::wallets::basic->wallet
+
+# Atomic Swap script: adds an asset from the note into consumers account and
+# creates a note consumable by note issuer containing requested ASSET.
+#
+# Requires that the account exposes: 
+#
+# Inputs: [ASSET, serial_num]
+# Outputs: []
```

Comment:
I don't think this is correct. Inputs into this script is just the note's script root (which we drop on line 24 below). So, this should be:
```
# Inputs: [SCRIPT_ROOT]
# Outputs: []
```